amundsen_application/static/js/ducks/dashboard/sagas.ts

```diff
@@ -8,8 +8,8 @@ import { GetDashboard } from './types';
 export function* getDashboardWorker(action): SagaIterator {
   try {
     const uri = action.payload.uri;
-    const dashboard = yield call(API.getDashboard, uri);
-    yield put(getDashboardSuccess(dashboard));
+    const { dashboard, statusCode } = yield call(API.getDashboard, uri);
```

Comment:
Is `statusCode` actually available on this result? If I'm reading this correctly this will be the `response.data` from the request and my understanding is that the status code would be found at `response.status`.

Reply:
Good catch. Will fix.